test/e2e/testdata/meteringconfigs/node-selector-prometheus-importer-disabled.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: metering.openshift.io/v1
```

Comment:
So this is meant to be a Metering CR with as many options used as possible?

Reply:
It's just the common MeteringConfig we use throughout the test installs + a node selector configuration.